spaces/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+
+from .models import Address, Place
+
+
+class AddressSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Address
+        fields = ('street_name', 'civic', 'floor', 'door', 'city', 'postal_code', 'region', 'nation')
+
+
+class PlaceSerializer(serializers.HyperlinkedModelSerializer):
+    address = AddressSerializer()
+
+    class Meta:
+        model = Place
+        fields = ('url', 'name', 'owner', 'description', 'address')
+
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get('name', instance.name)
+        instance.owner = validated_data.get('owner', instance.owner)
+        instance.description = validated_data.get('nationality', instance.description)
+        instance.address = validated_data.get('address', instance.address)
+        instance.save()
+        return instance
+
+    def create(self, validated_data):
+        address_data = validated_data.get('address')
+        address = Address.objects.create(street_name=address_data.get('street_name'),
```

Comment:
It seems working without the explicit implementation of .update(). So we can immediately remove that code.
However, according to my experience, if we don't explicitly implement the .create() method we run into the following error:

`AssertionError: The `.create()` method does not support writable nested fields by default.
Write an explicit `.create()` method for serializer `spaces.serializers.PlaceSerializer`, or set `read_only=True` on nested serializer fields.`